android/build.gradle

```diff
@@ -27,7 +27,7 @@ def shouldUseCommonInterfaceFromReanimated() {
         def json = new JsonSlurper().parseText(inputFile.text)
         def reanimatedVersion = json.version as String
         def (major, minor, patch) = reanimatedVersion.tokenize('.')
-        return Integer.parseInt(minor) >= 3
+        return Integer.parseInt(major) >= 2 && Integer.parseInt(minor) >= 3
```

Comment:
What about Reanimated 3.0? 🧐 😄

Reply:
Fair point, I'll leave that for future me to solve.